Set the default behavior on alpha to use -mieee since the large majority of
time we want this (bad/weird things can happen with packages built without
-mieee).

To satisfy those people who may not want -mieee forced on them all the time,
we also provide -mno-ieee.

Patch by Mike Frysinger <vapier@gentoo.org>

--- gcc-4_1-branch/gcc/config/alpha/alpha.h
+++ gcc-4_1-branch/gcc/config/alpha/alpha.h
@@ -96,7 +96,7 @@
   while (0)
 #endif
 
-#define CPP_SPEC "%(cpp_subtarget)"
+#define CPP_SPEC "%(cpp_subtarget) %{!no-ieee:-mieee}"
 
 #ifndef CPP_SUBTARGET_SPEC
 #define CPP_SUBTARGET_SPEC ""
--- gcc-4_1-branch/gcc/config/alpha/alpha.opt
+++ gcc-4_1-branch/gcc/config/alpha/alpha.opt
@@ -40,7 +40,7 @@
 Request IEEE-conformant math library routines (OSF/1)
 
 mieee
-Target Report RejectNegative Mask(IEEE)
+Target Report Mask(IEEE)
 Emit IEEE-conformant code, without inexact exceptions
 
 mieee-with-inexact
